peers.js

```diff
@@ -33,7 +49,7 @@ async function start() {
         const promises = [];
         for (let k = 0; k < peers.length; k++) {
             if (peers[k].startsWith('127.0.0')) continue;
-            promises.push(execPromise('ping ' + peers[k].split(':')[0] + ' -c 1 -w 2'));
+            promises.push(axios.get('http://' + peers[k] + '/block/height/770811').catch(function(err){return err}))
```

Comment:
what does '/block/height/770811' mean?

Reply:
it's a direct peer api request to download block 770811, which is just a random block I was using